waffle_hub/hub/adapter/autocare_dlt/autocare_dlt_hub.py

```diff
@@ -192,7 +192,7 @@ def get_metrics(self) -> list[dict]:
         return metrics
 
     # Train Hook
-    def on_train_start(self, cfg: TrainConfig):
+    def _on_train_start(self, cfg: TrainConfig):
```

Comment:
Since this is a callback function, it would be better to leave out the "_" and keep the previous code.

Reply:
We can choose one solution between "adding `_` to the method" or "restrict".
Which one do you prefer @Jiyong-Jeon ?
I think the second solution might be better.

Reply:
When a function is intended for internal use only and should not be used externally, I added '_' to ensure visibility.
If there is even a slight possibility of it being used externally, I applied a 'restrict'.
I think the '_' format is necessary, so I will adjust and modify the function to be applied as 'restrict' and the function to be applied as '_'.